manager/controllers/advisories.go

```diff
@@ -90,6 +91,7 @@ func AdvisoriesListHandler(c *gin.Context) {
 		// Error handling and setting of result code & content is done in ListCommon
 		return
 	}
+	mergeFiltersMaps(meta.Filter, filters)
```

Comment:
We could simplify that `query, meta, links, err := ListCommon(query, c, filters, "/api/patch/v1/advisories", AdvisoriesOpts)`